javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,138 @@
+//Item array removal
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ]; 
+  const nameToRemove = "Ahmad";
+  for(let arrayIndex of names ){
+    if(nameToRemove === arrayIndex){
+        names.splice(arrayIndex ,1)    
```

Comment:
This removed "Peter", you need to remove "Ahmad".

```
Remaining array after removel:  Ahmad,Yana,kristina,Rasmus,Samuel,katrine,Tala
```
The problem is this `names.splice(arrayIndex ,1)`. `arrayIndex` holds the value "Ahmad". Instead, you need to get the array index where "Ahmad" is. Try this.
```
const index = names.indexOf(nameToRemove);
names.splice(index, 1);
```